lib/elixir_boilerplate_graphql/elixir_boilerplate_graphql.ex

```diff
@@ -19,6 +19,11 @@ defmodule ElixirBoilerplateGraphQL do
 
     config
     |> Absinthe.Plug.default_pipeline(options)
```

Comment:
Should we revisit our approach fixing the `token_limit` to use a similar approach in one of the pipeline phases?

https://github.com/mirego/elixir-boilerplate/blob/5e56e7aa462f9aed5f2bb38d91ea6920f319505b/lib/elixir_boilerplate_graphql/elixir_boilerplate_graphql.ex#L26-L33

Reply:
`token_limit` is an [Absinthe setting](https://hexdocs.pm/absinthe/complexity-analysis.html#token-limits).